src/dotnet/CarbonAware.DataSources.WattTime/src/Configuration/ServiceCollectionExtensions.cs

```diff
@@ -1,13 +1,20 @@
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.DependencyInjection.Extensions;
+using Microsoft.Extensions.Configuration;
 using CarbonAware.Interfaces;
+using CarbonAware.Tools.WattTimeClient.Configuration;
+
 
 namespace CarbonAware.DataSources.WattTime.Configuration;
 
 public static class ServiceCollectionExtensions
 {
     public static void AddWattTimeDataSourceService(this IServiceCollection services)
     {
```

Comment:
Nit pick and I am also unaware with the coding conventions in C#, so ignore if this seems irrelevant. Do we need to create vars or can the code block be simplified by reducing it to a line -
services.ConfigureWattTimeClient(new ConfigurationBuilder().Build());

Reply:
Yup, it can be done in oneline too. I'll keep this way since it requires potentially to set env vars and/or config file.